### ISSUE TITLE
Extract the cornernet head from the whole network

### ISSUE THREAD

**Issue by ratnajitmukherjee** [feifeiwei/Pytorch-CornerNet#2]
Hey, 

Can you kindly provide me a guideline as to how I can just extract the cornernet head from the whole network. I want to extract the corner pooling layrs and the loss functions without using the backbone. 

Sincerely,
Ratnajit
